### PR TITLE
companies.update: add custom fields

### PR DESCRIPTION
**Pull request by dietrichm**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -929,6 +929,22 @@ Update a company.
...
+                    + value: `foo`,`bar` (array[string]) - For multiple selection fields
+                    + value: true (boolean) - For Yes/No fields
+                    + value (object) - For related Teamleader objects
+                        + type: `company` (enum[string])
+                            + Members
+                                + company
+                                + contact
+                                + product
+                                + user
+                        + id: `bf6765de-56eb-40ec-ad14-9096c5dc5fe1` (string)
```
Comment (WouterSioen) [inline apiary.apib]: Q: since this is a copy paste, wouldn't it be better to add a `CustomFieldValue` datastructure?

**Review by WouterSioen — DISMISSED** [review]

File: apiary.apib (hunk 1 above)
Comment (dietrichm) [reply apiary.apib]: Done!

 :ok: by @dietrichm on 04/05/2018 13:14:30

**Review by WouterSioen — APPROVED** [review]

**Review by mansilaxel — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Review by mrtus — APPROVED** [review]
Looks good 👍 But linting is failing

**Comment by dietrichm** [comment]
Linting fails due to a node module. 😞 

```
Error: The module '/tmp/build/791ac344/git-api/node_modules/protagonist/build/Release/protagonist.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 59. This version of Node.js requires
NODE_MODULE_VERSION 64. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
```

**Review by mrtus — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]

**Pull request merged**